vnext/Microsoft.ReactNative/ReactInstanceSettings.idl

```diff
@@ -12,129 +12,199 @@ import "RedBoxHandler.idl";
-Non-compatible changes still cause full reloads.\n\
-See [Fast Refresh](https://reactnative.dev/docs/fast-refresh) for more information on Fast Refresh.")
+    DOC_STRING(
+      "Should the instance trigger the hot module reload logic when it first loads the instance.\n"
+      "Most edits should be visible within a second or two without the instance having to reload.\n"
+      "Non-compatible changes still cause full reloads.\n"
+      "See [Fast Refresh](https://reactnative.dev/docs/fast-refresh) for more information on Fast Refresh.")
     Boolean UseFastRefresh { get; set; };
 
-    DOC_STRING("Enable live reload to load the source bundle from the React Native packager.\n\
-When the file is saved, the packager will trigger reloading.\n\
-**For general use this has been replaced by @.UseFastRefresh.**")
+    DOC_STRING(
+      "Enable live reload to load the source bundle from the React Native packager.\n"
+      "When the file is saved, the packager will trigger reloading.\n"
+      "**For general use this has been replaced by @.UseFastRefresh.**")
     Boolean UseLiveReload { get; set; };
 
-    DOC_STRING("Enables debugging in the JavaScript engine (if supported).  \n\
-For Chakra this enables you to debug the JS runtime directly within your app using Visual Studio -> Attach to process (Script)")
+    DOC_STRING(
+      "Enables debugging in the JavaScript engine (if supported).\n"
+      "For Chakra this enables you to debug the JS runtime directly within your app using "
+      "Visual Studio -> Attach to process (Script)")
     Boolean UseDirectDebugger { get; set; };
 
-    DOC_STRING("For direct debugging, whether to break on the next line of JavaScript that is executed.  This can help debug issues hit early in the JavaScript bundle load.\n\
-***Note: this is not supported with the Chakra JS engine which is the currently used JavaScript engine***")
+    DOC_STRING(
+      "For direct debugging, whether to break on the next line of JavaScript that is executed.\n"
+      "This can help debug issues hit early in the JavaScript bundle load.\n"
+      "***Note: this is not supported with the Chakra JS engine which is the currently used JavaScript engine***")
     Boolean DebuggerBreakOnNextLine { get; set; };
 
     DOC_STRING("Flag controlling whether the JavaScript engine uses JIT compilation.")
     DOC_DEFAULT("true")
     Boolean EnableJITCompilation { get; set; };
 
-    DOC_STRING("For JS engines that support bytecode generation, this controls if bytecode should be generated when a JavaScript bundle is first loaded.\n\
-Subsequent runs of the application should be faster as the JavaScript will be loaded from bytecode instead of the raw JavaScript.  \n\
-@.ByteCodeFileUri must be set to a location the application has write access to in order for the bytecode to be successfully cached.")
+    DOC_STRING(
+      "For JS engines that support bytecode generation, this controls if bytecode should be generated when "
```

Comment:
which are the JS engines that support bytecode generation? (point to JS / Hermes doc)

Reply:
It does not really work today outside of Office. Hermes does not support the bytecode generation at runtime. They do it at build time.

Reply:
I added there that it does not work today for UWP apps.